_data/projects/nfsw-server.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: Need For Speed World Server
+desc: Just as the name suggests, this is an open source server for the game, compatible with the "SBRW" (Soap Box Race World) game launcher.
+site: https://github.com/aadityachakravarty/sbrw-server
+tags:
+- javascript
+- racing
+- backend
+- nodejs
+- nfsworld
+- sbrwserver
+- sbrw
+upforgrabs:
+  name: hobby project
+  link: https://github.com/aadityachakravarty/sbrw-server/labels/up-for-grabs
```

Comment:
There aren't any issues listed on the repository for new contributors to look at.

Is that something you'd be able to setup before I merge this?

Reply:
Added a few to begin with, will add more as the project goes on.